### PR TITLE
Odin protocol assets, RPC and REST API endpoints update

### PR DESCRIPTION
**Pull request by Satchitananda**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
   - The `description` and/or `extended_description` of the asset in the Chain Registry is informative.
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)
   - Note that it is recommended to include an X (fka, Twitter) profile URL with each asset.

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: [Odin/Osmosis Liquidity Pool](https://app.osmosis.zone/pool/777)
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -940,6 +940,18 @@
       "path": "transfer/channel-258/mO9W",
       "osmosis_verified": false
     },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "osmosis_verified": false
+    },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:948]: Please add "osmosis_unlisted": true 
to both assets.
Also please move this addition to the bottom of the list

File: osmosis-1/osmosis.zone_assets.json [hunk 2]
```diff
@@ -940,6 +940,20 @@
...
       "osmosis_verified": false
     },
+    {
+      "chain_name": "odin",
+      "base_denom": "udoki",
+      "path": "transfer/channel-258/udoki",
+      "osmosis_verified": false,
+      "osmosis_unlisted": true
+    },
+    {
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:950]: Please move this addition to the bottom of the list

**Comment by JeremyParish69** [comment]
Which pools have MYRK and DOKI?

**Comment by AddySiz** [comment]
> Which pools have MYRK and DOKI?

no pool set up for MYRK & DOKI yet. PLan to set up for DOKI asap

File: osmosis-1/osmosis.zone_assets.json (hunk 2 above)
Comment (Satchitananda) [reply osmosis-1/osmosis.zone_assets.json:950]: ok! Will do

Comment (Satchitananda) [reply osmosis-1/osmosis.zone_assets.json]: This one is fixed

**Comment by Satchitananda** [comment]
@JeremyParish69 I have reverted changes to assets, can we merge RPC and API addresses changes to resolve CORS issue?

**Pull request merged**